streams-tests/jvm/src/test/scala/zio/stream/ToIteratorSpec.scala

```diff
@@ -0,0 +1,32 @@
+package zio.stream
+
+import zio.ZIOBaseSpec
+import zio._
+import zio.test.Assertion.equalTo
+import zio.test._
+
+object ToIteratorSpec extends ZIOBaseSpec {
```

Comment:
Why don't we just move this into `StreamSpec` at this point since it is not that large to avoid creating another file? May also want to inline `testMM` since we are only using it one place and just call `.use(ZIO.succeed)` at the end